stdlib/ocaml/pprint.mc

```diff
@@ -7,7 +7,8 @@ include "name.mc"
 include "intrinsics-ops.mc"
 
 let _isValidChar = lam c.
-  or (isAlphanum c) (or (eqChar c '_') (eqChar c '\''))
+  if isAlphanum c then true
+  else eqChar c '\''
```

Comment:
The old version accepts `'_'`, should the new version also do that?

Reply:
The reason I removed it is because we escape invalid characters with `'_'`, so it's saving one extra computational step. And as far as I know that's the only thing we're using this function for.